renderdoc/driver/dxgi/dxgi_common.cpp

```diff
@@ -1391,7 +1391,8 @@ DXGI_FORMAT MakeDXGIFormat(ResourceFormat fmt)
         break;
       case eSpecial_S8: RDCERR("D3D has no stencil-only format"); break;
       case eSpecial_D16S8: RDCERR("D3D has no D16S8 format"); break;
-      default: RDCERR("Unrecognised/unsupported special format %u", fmt.specialFormat); break;
+      default:
+        break;    // Return DXGI_FORMAT_UNKNOWN to convert texture.
```

Comment:
The idea of this change is for *all* unsupported formats to return DXGI_FORMAT_UNKNOWN so that they can be remapped. We shouldn't pick and choose formats to remap since all that matters for the proxying driver is that it's not supported. Maybe in some API a 3-byte 1-channel texture is perfectly valid. The replaying driver is then responsible for rendering it out to some common format that will be supported.

YUV has the problem that the special format enum aliases about a dozen different video formats all to eSpecial_YUV. For now we can just return UNKNOWN for YUV without printing an error, and the remapping code will error because it doesn't know how to remap it. That's fine for now, we can add further remaps later.

There's a similar problem for ASTC where the ResourceFormat doesn't contain e.g. block size, but this would only be a problem if the desktop proxy supported ASTC natively and wanted to create an accurate proxy texture.

I'm going to merge this PR in now rather than go for another round of changes, I'll fix this up in a post-merge commit.